subprojects/platform-native/platform-native.gradle

```diff
@@ -26,6 +26,7 @@ dependencies {
     integTestRuntime project(':ideNative')
 
     testFixturesCompile project(':internalIntegTesting')
+    testFixturesCompile project(':languageNative')
```

Comment:
We shouldn't add a cycle between the projects. This says that something is in the wrong project.

Reply:
I removed this in preference to keeping the compiler version on the `InstalledToolChain` instead.